### DISCUSSION TITLE
0.4.2版本的db数据库好像无法导入到0.7.2本的memos

### DISCUSSION THREAD

**Discussion by leoneoo** [usememos/memos#485] [Q&A] [answered]
0.4.2版本的db数据库好像无法导入到0.7.2本的memos？请问有办法解决吗

**Comment by boojack** [comment, marked as answer]
数据库文件先做好备份，然后直接将系统更新至最新的版本。如果遇到报错，麻烦提供下相关错误信息。

**Reply by leoneoo** [reply]
请问一下，如果备份了数据库文件，但原网站崩溃了，无法复原。保留的数据库文件，能否直接拷贝到相同配置的其他服务器部署的memos中。

**Reply by boojack** [reply]
> 但原网站崩溃了，无法复原

如果想继续使用最新版，可以提供下崩溃的错误信息？

> 保留的数据库文件，能否直接拷贝到相同配置的其他服务器部署的memos中。

数据库文件可以直接在相同版本的服务中运行，比如 0.4.2 的备份数据库文件，可以运行在 0.4.x 的 memos 服务中。

**Reply by leoneoo** [reply]
TKS已经解决问题了。Memso没有问题，应该是我数据库密码的问题。总之这个项目非常棒，以后可以备份数据库。但最好有个导出功能，可以使其他注册用户自己导出并备份数据！！！

**Reply by boojack** [reply]
> 最好有个导出功能

系统运行所产生的所有数据包括图片都在一个数据库文件中，不再需要所谓的“导出”。

> 其他注册用户自己导出并备份数据

这种情况建议其他注册用户自行部署。
